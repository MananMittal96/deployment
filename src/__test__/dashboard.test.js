import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import Dashboard from '../components/dashboard';

configure({ adapter: new Adapter() });

describe('MyComponent', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Dashboard />);
  
    expect(component).toMatchSnapshot();
  });
});