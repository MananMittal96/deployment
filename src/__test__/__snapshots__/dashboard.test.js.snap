// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `
<div
  className="dashboard container"
>
  <div
    className="row"
  >
    <div
      className="col s12 m6"
    >
      <ProjectList />
    </div>
  </div>
</div>
`;
